client/state/sites/plans/selectors.js

```diff
@@ -82,13 +82,11 @@ export function getPlanDiscountedRawPrice(
 	state,
 	siteId,
 	productSlug,
-	{
-		isMonthly = false
-	} = {}
+	{ isMonthly = false } = {}
 ) {
 	const plan = getSitePlan( state, siteId, productSlug );
 
-	if ( get( plan, 'rawPrice', -1 ) < 0 || get( plan, 'rawDiscount', -1 ) <= 0 ) {
+	if ( get( plan, 'rawPrice', -1 ) < 0 || get( plan, 'rawDiscount', -1 ) < 0 ) {
```

Comment:
Can we use the new `isPlanDiscounted` selector here instead of `get( plan, 'rawDiscount', -1 ) < 0`?